docs/pages/careers/staff-engineer-pigment-css.md

```diff
@@ -0,0 +1,126 @@
+
+For additional details about the culture, you can check our [careers](https://mui.com/careers/) and [about](https://mui.com/about/) pages and also our [public Handbook](https://mui-org.notion.site/Handbook-f086d47e10794d5e839aef9dc67f324b).
+
+## Why we're hiring
+
+The Pigment CSS team is just starting to take shape now, and we have big goals for the next few years. We need experienced people to work in an IC capacity, to research, spec, build, document, and ship high-quality, zero-runtime CSS-in-JS library with a focus on performance and great developer experience.
+
+Overall, both our open-source community and our premium products are growing fast (x2 YoY).
+We need talented people to keep that going!
+
+### Why this is interesting
+
+Our products empower React developers to build awesome applications faster – we see millions of developers on MUI's docs every year, one million a month.
+
+The core components that will depend on this library are widely used, you will receive a lot of feedback on your work. The community has high expectations, they will push you to do better, every day.
+The [results of the latest survey](https://mui.com/blog/2021-developer-survey-results/) can give you a great sense of where we will need to push the open-source library in the coming months.
+
+## The role
+
+### What you'll do on a day-to-day basis
+
+Depending on the day, you'll:
+
+- **Build**. You will provide solutions for integrating the library with various bundlers and frameworks. You will build new features and improve the DX of the library.
+- **Nurture community contributions**. You will provide guidance and direction to unlock the contributions of the community. Your time will often be way better spent doing this than fixing the problems yourself.
+- **Shape the product**. You will be laser-focused on the end goal. It's not about solving technical challenges but about the problem solved for the users.
+- **Keep technical debt in check**. You will make sure we can keep shipping features at a reasonable pace, align the team on "one way" of doing things and make sure engineers follow the conventions.
+- **Be a solver**. You will dig deep into arbitrarily complex problems and find an appropriate path forward. Some focus on a given area for long periods, others bounce from hotspot to hotspot as guided by organizational leadership.
+
+### The best parts of this job
+
+- **You'll be at the cutting edge of application development** — working on one of the fastest-growing CSS-in-JS libraries on the market.
+- **You'll be part of an active, open, friendly community** of developers that are excited about building awesome applications.
+- **Your role will be key to making MUI the go-to UI solution** for building applications, websites, and design systems that does not have to trade performance for a great developer expirience.
+
+### The worst parts of this job
```

Comment:
I have removed these sections from the Docs-infra role — I feel like we could be simpler and more to-the-point: https://mui.com/careers/react-engineer-docs-infra/